desk/app/chat.hoon

```diff
@@ -920,6 +962,24 @@
       (cu-post-notice for.delta '' ' was uninvited from the chat')
     ==
   ::
+  ++  cu-remark-diff
+    |=  diff=remark-diff:c
+    ^+  cu-core
+    =.  remark.club
+      ?-  -.diff
+        %watch    remark.club(watching &)
+        %unwatch  remark.club(watching |)
+        %read-at  !! ::  cu-core(last-read.remark.chat p.diff)
```

Comment:
confirming it's expected to bail here with zapzap?

Reply:
we don't use read-at anymore I don't think so yeah it's fine for now just need to remove at some point